docs/cheat-sheets/javascript-code-injection.mdx

```diff
@@ -0,0 +1,58 @@
+---
+slug: javascript-code-injection
+---
+
+import LinkToRegistryRule from "/src/components/LinkToRegistryRule"
+import CodeInjectionIntro from "/src/components/concept/_code-injection-intro.mdx"
+
+# Code injection prevention for JavaScript
+
+<CodeInjectionIntro />
+
+## 1. Executing or evaluating code
+
+### 1.A. running code with `vm` module
+
+The `vm` module enables compiling and running code within V8 Virtual Machine contexts. The `vm` module is not a security mechanism. Do not use it to run untrusted code. If code passed to vm functions is controlled by user input it could result in command injection. 
```

Comment:
```suggestion
The `vm` module enables compiling and running code within V8 Virtual Machine contexts. The `vm` module is not a security mechanism. Do not use it to run untrusted code. If code passed to vm functions is controlled by user input it could result in code injection. 
```